src/components/pagination/pagination.twig

```diff
@@ -5,7 +5,12 @@
   <{{ label_tag_name }} id="{{ label_id }}" class="u-hidden-visually">Pagination</{{ label_tag_name }}>
   <ul class="c-pagination__items" role="list">
     {% for page in pagination.pages %}
-      <li class="c-pagination__item">
+      <li
+        class="c-pagination__item"
+        {% if page.title == '&hellip;' or page.title == '…' %}
```

Comment:
This may be overkill, but should we check to make sure the `page` lacks a `link` just to be extra sure it isn't meaningful?

```twig
        {% if (not page.link) and (page.title == '&hellip;' or page.title == '…') %}
```

Or alternatively we could move the `li` into the conditional blocks. We're already repeating the `a` so it's not that big a deal to also repeat the `li`.

Reply:
Good call! I'll look into it now. 🔍 👀

Reply:
[Udpated!](https://github.com/cloudfour/cloudfour.com-patterns/pull/1983/commits/2fb1a4464bb458b41bec7ac71705d8847ac3265e)